starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -827,9 +828,16 @@ void reportFullyDrawn() {
   public void setCrashContext(String key, String value) {
     Log.i(TAG, "setCrashContext Called: " + key + ", " + value);
     crashContext.put(key, value);
+    if (this.handler) {
+      this.handler.onUpdate();
```

Comment:
Should we pass the new key/value here too? Or are you expecting we just call "getKimonoContext" in KimonoContextProviderImpl on ever onUpdate?

Reply:
Looking closer at the internal publisher code, I think this is code. We just need some signal, any signal, the crash context changes. Then it can re-run getKimonoContext.

Reply:
Also, let's just verify with Zhicheng on this before we commit it

Reply:
It is verified. I will submit at noon if there is no objection.